### PR TITLE
[MI-241] Update if statement on getConnectionStatus

### PR DESCRIPTION
**Pull request by joseconsador**
Adds logging when an exception is caught during Zendesk_Zendesk_Model_Api_Abstract::_call() . 
Also improves the conditional statement which caused the notice.

Updated the error message as per https://support.zendesk.com/agent/tickets/1134038

screen shot 2015-09-15 at 2 19 13 pm

/cc @miogalang @jwswj @mmolina @iandjx 
### References
- Jira link: https://zendesk.atlassian.net/browse/MI-241
### Risks
- Medium: Other exceptions may go through Zendesk_Zendesk_Model_Api_Abstract::_call()

### CODE REVIEW

**Comment by miogalang** [comment]
:+1:

File: src/app/code/community/Zendesk/Zendesk/Helper/Data.php [hunk 1]
```diff
@@ -350,16 +350,16 @@ public function getFormatedDateTime($dateToFormat) {
...
                 return array(
                     'success'   => true,
                     'msg'       => Mage::helper('zendesk')->__('Connection to Zendesk API successful'),
                 );
             }
-            
+
             $error = Mage::helper('zendesk')->__('Connection to Zendesk API failed') .
-                '<br />' . Mage::helper('zendesk')->__('Troubleshooting tips can be found at <a href=%s>%s</a>', 'https://support.zendesk.com/entries/26579987', 'https://support.zendesk.com/entries/26579987');
+                '<br />' . Mage::helper('zendesk')->__("Click 'Save Config' and try again. If the issue persist, check if the entered Agent Email Address and Agent Token combination is correct.");
```
Comment (jwswj) [inline src/app/code/community/Zendesk/Zendesk/Helper/Data.php:376]: Translation will also need to be updated

File: src/app/locale/en_US/Zendesk_Zendesk.yml [hunk 2]
```diff
@@ -224,7 +224,7 @@ parts:
   - translation:
         key: "text.integrations.magento.config.test_connection.tips"
         title: "Configuration message: suggestion to look at provided URL for troubleshooting"
-        value: 'Troubleshooting tips can be found at <a href="%s" target="_blank">%s</a>'
```
Comment (ydrain) [inline src/app/locale/en_US/Zendesk_Zendesk.yml:227]: When you need to modify the text,  please create a new key and mark this as `obsolete`:

https://zendesk.atlassian.net/wiki/display/I18N/Editing+strings

Comment (joseconsador) [inline src/app/locale/en_US/Zendesk_Zendesk.yml:227]: thanks :haircut:

**Comment by ydrain** [comment]
:+1: for strings

**Comment by jwswj** [comment]
:+1:

**Comment by pdeuter** [comment]
This PR has 2 +1s, so I will merge now so that the string update will get into this week's project.
If this PR is not ready, please revert.

Going forward, please separate the string change into a separate PR so that PR can be merged in advance of the code PR.  That way, the translation can be in progress while the code is still being perfected.

-Paul

**Pull request merged**